### PR TITLE
chore: migrate to `vite-bundle-analyzer`

### PR DESCRIPTION
**Pull request by nonzzz**
---

This version 

image

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -205,36 +209,6 @@ importers:
...
-      less:
-        specifier: ^4.1.3
-        version: 4.1.3
-      react:
-        specifier: ^18.2.0
-        version: 18.2.0
-      sass:
-        specifier: ^1.59.3
-        version: 1.67.0
-
```
Comment (NWYLZW) [inline pnpm-lock.yaml:237]: This doesn't look like it should be submitted.

File: vite.config.ts [hunk 2]
```diff
@@ -119,7 +119,7 @@ export default defineConfig(async env => ({
         }
       }
     }),
-    env.mode === 'production' ? visualizer() : undefined,
+    analyzer({ reportFileName:'stats.html' }),
```
Comment (NWYLZW) [inline vite.config.ts:122]:
```suggestion
    analyzer({ reportFileName: 'stats.html' }),
```

File: vite.config.ts [hunk 3]
```diff
@@ -119,7 +119,7 @@ export default defineConfig(async env => ({
         }
       }
     }),
-    env.mode === 'production' ? visualizer() : undefined,
```
Comment (NWYLZW) [inline vite.config.ts:122]: I thought maybe we don't need that plugin in the dev environment?

Comment (nonzzz) [reply vite.config.ts:122]: Btw vite's development mode only in server mode. And this plugin only work at build stage

File: pnpm-lock.yaml (hunk 1 above)
Comment (nonzzz) [reply pnpm-lock.yaml]: I notice `external/zodui` is a part of this project. So i install the submodule and it's work well :)

**Pull request merged**